### DISCUSSION TITLE
Doesn't set `$pwd` properly on Windows

### DISCUSSION THREAD

**Discussion by brian6932** [antonmedv/llama#135] [General]
Returned `m.path` has mixed `/` & `\`, but that's not the cause of this, seems like another field.
This is specifically about:
```
Esc, q        Exit with cd
```

**Comment by antonmedv** [comment]
Did you add function from readme?

**Reply by brian6932** [reply]
Yes, but I forgot about it tbh, since it didn't have the same name. My bad.
Maybe something like this would be nice
```pwsh
function walk() {
	Set-Location (& (Get-Command -CommandType Application walk) $args)
}
```
